README.md

```diff
@@ -2,7 +2,7 @@
 
 # Developer's Portfolio of Philip Modinos
 
-> This is the professional developer portfolio of Philip Modinos.  Philip has been coding since 2019, focusing on html/css and php/mysql using XAMPP.  He has developed several sites for German music companies "Opera Classica Europa" and "Bad Ems Music Festival" including the site www.offenbachgp.com, which was an online singing competition, that included applause and comments by the audience and a fully automated registration system.  
+> This is the professional developer portfolio of Philip Modinos.  You can visit this website at:  https://ninjaphil24.github.io/portfolio_1/ <br>Philip has been coding since 2019, focusing on html/css and php/mysql using XAMPP.  He has developed several sites for German music companies "Opera Classica Europa" and "Bad Ems Music Festival" including the site www.offenbachgp.com, which was an online singing competition, that included applause and comments by the audience and a fully automated registration system.  
```

Comment:
- [ ] **Optional:** Use the **_Live Demo_** subheader as specified on the [Microverse template](https://github.com/microverseinc/readme-template) to always document links to your demo page on the README.

Reply:
OK, I understand now.  Apologies.  It's done.